### PR TITLE
feat(converter): add infoSummaryRefractorPlugin

### PR DESCRIPTION
**Pull request by kowalczyk-krzysztof**
This plugin removes the `summary` fixed field of `Info Object`.

Refs SWG-9954

### CODE REVIEW

File: packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/info-summary.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+type InfoSummaryPluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const infoSummaryRefractorPlugin =
+  ({ annotations }: InfoSummaryPluginOptions) =>
+  () => {
+    const annotation = new AnnotationElement(
+      'The "summary" field of Info Object is not supported in OpenAPI 3.0.3. It will be removed from the converted document.',
```
Comment (char0n) [inline packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/info-summary.ts:12]: We're using past tense for annotations, describing things that already happened.

```suggestion
      'The "summary" field of Info Object is not supported in OpenAPI 3.0.3. It has been removed from the converted document.',
```

File: packages/apidom-converter/test/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/info-summary/index.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  toJSON,
+  includesClasses,
+} from '@swagger-api/apidom-core';
+
+import convert from '../../../../../src';
+
+describe('converter', function () {
+  context('strategies', function () {
+    context('openapi-3-1-to-openapi-3-0-3', function () {
+      context('info summary', function () {
```
Comment (char0n) [inline packages/apidom-converter/test/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/info-summary/index.ts:17]:
```suggestion
      context('info-summary', function () {
```

**Review by char0n — CHANGES_REQUESTED** [review]
Looks good! Two small nitpicks detected.

**Comment by char0n** [comment]
@kowalczyk-krzysztof notice I've change the commit msg and it's body. 

Would be ideal if this PR will eventually gets sqashed+merged to following commit msg:

```
feat(converter): add infoSummaryRefractorPlugin

This plugin removes the summary fixed field of Info Object.

Refs #3697
```

The commit message header is consistent with the previous ones - `feat(converter): add infoSummaryRefractorPlugin`

The commit body explains what this plugin is actually about.

The commit footer references the public GitHub epic associated with converter: https://github.com/swagger-api/apidom/issues/3697

**Comment by kowalczyk-krzysztof** [comment]
> @kowalczyk-krzysztof notice I've change the commit msg and it's body.
> 
> Would be ideal if this PR will eventually gets sqashed+merged to following commit msg:
> 
> ```
> feat(converter): add infoSummaryRefractorPlugin
> 
> This plugin removes the summary fixed field of Info Object.
> 
> Refs #3697
> ```
> 
> The commit message header is consistent with the previous ones - `feat(converter): add infoSummaryRefractorPlugin`
> 
> The commit body explains what this plugin is actually about.
> 
> The commit footer references the public GitHub epic associated with converter: #3697

Thanks for the comment. I'll make sure to use this convention from now on.

**Review by char0n — APPROVED** [review]

**Comment by char0n** [comment]
The tests are failing. Probably other tests got affected by new plugin.

**Comment by kowalczyk-krzysztof** [comment]
> The tests are failing. Probably other tests got affected by new plugin.

It was the change from `info summary` to `info-summary` which caused the snapshot to become obsolete. I've pushed a fix.

**Comment by char0n** [comment]
@kowalczyk-krzysztof please don't rebase the PRs into single commits. By doing this we loose all the history of the PR and comments attached to specific commit changes.

GitHub has a squash and merge functionality directly available:

image

After clicking it allows you to modify entire commit message: 
image

**Pull request merged**